web/styles/popovers.css

```diff
@@ -895,14 +928,18 @@ ul {
 #giphy_grid_in_popover {
     /* 300px of GIPHY grid + 5px is the extra gutter space
      * between gif columns. */
-    width: 305px;
+    width: 307px;
     border: 0;
     padding: 0;
 
     .giphy-gif {
         cursor: pointer;
     }
 
+    .giphy-gif:focus {
+        outline: -webkit-focus-ring-color auto 1px;
```

Comment:
Ok, `-webkit-focus-ring-color` is a value I did not know about earlier. I gave [this article](https://remysharp.com/til/css/focus-ring-default-styles) a read, and I think we should also add the`Highlight` value to support Firefox.

Reply:
We also prefer the `[ width || style || color ]` ordering for the shorthand values.